flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/common/utils/SchemaTransform.java

```diff
@@ -74,7 +74,7 @@ private static Map<String, List<Schema.Type>> initializeBigQueryToAvroTypesMappi
         mapping.put("DATETIME", Arrays.asList(Schema.Type.STRING));
         mapping.put("TIME", Arrays.asList(Schema.Type.STRING, Schema.Type.LONG));
         mapping.put("JSON", Arrays.asList(Schema.Type.STRING));
-
+        // TODO: Add support for RANGE type.
```

Comment:
Instead of adding an explicit `TODO`, remove this line and track via a buganizer task. This repo is publicly visible.

Reply:
ACK.